src/data/memberships.json

```diff
@@ -4,5 +4,41 @@
     "destinationID": "default_membership",
     "destinationType": "room",
     "members": [{"personID": "default"}, {"personID": "self"}, {"personID": "call"}, {"personID": "presenting"}]
+  },
+  "remoteVideo": {
```

Comment:
Are most of these memberships being used? I only see `noMembers` used in the stories/test
We should remove them if we're not currently using them

Reply:
Every test and story gets the membership object for the meeting ID so I had to add all of them. It would be easier if the memberships adapter got the data from the meeting json instead but it doesn't.

Reply:
Got it, just wanted to understand since I'm making changes to the data in an upcoming PR. Can probably update that then